src/Components/NewBeer.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import NavBar from '../Components/NavBar'
+
+
+export default class NewBeer extends Component {
+
+    state = {
+        name: '',
+        description: '',
+        tagline: '',
+        first_brewed: '',
+        brewers_tips:'',
+        attenuation_level:'',
+        contributed_by:''
+    }
+
+    onChangeHandler = e => {
+        let { name, value } = e.target
+        this.setState({
+            [name]: value
+        })
+      }
+
+      submitHandler= (e) => {
+        e.preventDefault()
+        const {name, description, tagline, firstBrewed,  brewersTips, attenuationLevel, contributedBy} = e.target
```

Comment:
You already have the correct values of the inputs in the state, these are the values that you should send